packages/core/src/Stores/client-store.js

```diff
@@ -734,6 +734,7 @@ export default class ClientStore extends BaseStore {
                 runInAction(() => {
                     const new_account = Object.assign({}, this.accounts[this.loginid]);
                     new_account.currency = currency;
+                    if (!('balance' in new_account)) new_account.balance = 0;
```

Comment:
Could we fetch `balance` value for cryptos from the defaults set in **currency.js**? In particular, `fractional_digits`.

Reply:
OK as we discussed this is already handled in our `Money` component. You can ignore this.